lib/reek/cli/application.rb

```diff
@@ -48,6 +49,10 @@ def configure_app_configuration(config_file)
       def command_class
         options.generate_todo_list ? Command::TodoListCommand : Command::ReportCommand
       end
+
+      def argv
```

Comment:
I think it's needed by Files, but I think that whole module may just have to be moved here instead.